index.html

```diff
@@ -5,10 +5,21 @@
     <title>All-Inclusive-Web-Team</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap" rel="stylesheet">
   </head>
   <body>
-    <nav>
-      <img id='logo' src="assets/dcrdlogo.png" alt="logo">
-    </nav>
+    <div id="nav-bar">
+      <ul>
+        <li id="studentEl">Students</li>
+        <li id="teacherEl">Teachers</li>
+        <img id="logo" src="./assets/dcrdlogo.png">
+        <li id="founderEl">Founder</li>
```

Comment:
I would skip IDing these elements until there's a use case for differentiating them.

Reply:
I see `#logo` has CSS rules below, that's good enough reason to leave that one alone.